lexicons/app/bsky/feed/getTimeline.json

```diff
@@ -9,6 +9,11 @@
         "type": "params",
         "properties": {
           "algorithm": {"type": "string"},
+          "excludePostTypes": {
+            "type": "array",
+            "items": {"type": "string"},
```

Comment:
Current codegen doesn't support `knownValues` here, for a nested `string[]`. But, it might have been a skill issue.